db/migrate/20251112085230_add_diagnosed_confirmed_at_to_patients.rb

```diff
@@ -0,0 +1,16 @@
+class AddDiagnosedConfirmedAtToPatients < ActiveRecord::Migration[6.1]
+  def up
+    add_column :patients, :diagnosed_confirmed_at, :datetime
```

Comment:
I recommend delegating the diagnosis date to the medical conditions.

Reply:
This way, there's one source of truth for the diagnosis. Now, there's two. Patient and MedicalHistory